_docs/setup/kubernetes/aws-nlb.md

```diff
@@ -0,0 +1,92 @@
+---
+title: AWS NLB
+overview: Instructions to use and configure ingress istio with aws NLB.
+
+order: 12
```

Comment:
order doesn't look correct.  Could you use a higher number?  Typically newer features go towards the end of the Kubernetes quickstart, but I'm not sure there is a specific policy on ordering.  Rendering the landing page and attaching a screen shot may help 👍